designs/0016-hessian_optimize_cmdstan.md

```diff
@@ -13,7 +13,16 @@ When computing a MAP estimate, the Hessian of the log density can be used to con
 
 I have a Stan model that is too slow to practically sample all the time. Because optimization seem to give reasonable results, it would be nice to have the normal approximation to the posterior to give some sense of the uncertainty in the problem as well.
 
-An approximate posterior covariance comes from computing the inverse of the Hessian of the negative log density.
+It is standard to compute a normal approximation to the posterior covariance comes from the inverse of the Hessian of the negative log density.
+
+If the MAP estimate is ```u```, and the Hessian of the unnormalized log density at u is ```H```, then a posterior draw on the constrained scale is:
+
+```
+unconstrained_sample = multivariate_normal_rng(mean = u, cov = -inverse(H))
+constrained_sample = constrain(unconstrained_sample)
+```
+
+We can output unnormalized log densities of the actual model and the approximate model to compute importance sampling diagnostics and estimates.
```

Comment:
We can output unnormalized log densities in the unconstrained space of the true posterior and the approximate posterior